### ISSUE TITLE
Have rustdoc output lints for better docs

### ISSUE THREAD

**Issue by mgattozzi** [steveklabnik/rustdoc#77]
This is probably far off but it would be nice to lint doc output so that people can write better docs. For instance avoiding usage of words like "Simply" or "Just". This can help provide standardization of best practices and help guide the community towards higher quality documentation. Of course we should provide the options to turn off these items like any other linting tool.

I would also think that having it only on for building your own crate or binary would be good. Having lints from other projects show up during the build process is annoying.

**Comment by steveklabnik** [comment]
Marking as "Quest" because this is a big thing that someone will need to think through all the details of.

**Comment by mgattozzi** [comment]
Yeah this is by no means an easy thing at all and I wouldn't expect this to be done till way after the basic functionality is implemented.

**Comment by steveklabnik** [comment]
Checking for stuff like https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md would be 👍 too

**Comment by mgattozzi** [comment]
I think #78 is also tangentially related to this.

**Comment by steveklabnik** [comment]
Marking this as hard because there's a lot of work in determining what the lints should be, not just implementing them.
